gamemodes/terrortown/gamemode/client/cl_equip.lua

```diff
@@ -995,17 +999,12 @@ local function ReceiveBought()
 	local num = net.ReadUInt(8)
 
 	for i = 1, num do
-		local s = net.ReadString()
-		if s ~= "" then
-			client.bought[#client.bought + 1] = s
+		local equipmentId = net.ReadString()
+		if equipmentId ~= "" then
+			client.bought[#client.bought + 1] = equipmentId
```

Comment:
```suggestion
		local equipmentClass = net.ReadString()
		if equipmentClass ~= "" then
			client.bought[#client.bought + 1] = equipmentClass
```

Reply:
Hmm, not sure this is an improvement, but I don't care that much. But then you should add this to the docs here as well: 
![image](https://github.com/TTT-2/TTT2/assets/13639408/34d3e966-c0c3-40b7-85de-ee40c3c24077)

Something like ` @param string equipmentId The class name of the equipment used as identifier`

Otherwise it is not really clear what a "equipmentId" imho

Reply:
Its like last time, I guess. It's either obvious or should not be used.
But currently all equipments equal in `classname = id = name`.

So I wouldn rather not add a description everywhere for something that should be known.

But I just took a look at the rest of our codebase and we only use `equipmentId` in the newest shop rework parts 1-3.

We use `.id` to check nearly everything shop- relevant. And to give items we use `cls` for `classname`.

So either we agree on changing everything to `id` or revert back to `classname`.
Im just not sure why in the first place we used `.id` and not `.name`.

Reply:
I think name is pretty fitting as well. Even though it is a bit conusing to me why we call something "name" that is derived from a class. I guess this is just what happens when 100 people work on the same code over years.

I don't have a good solution here. The thing I don't like about id is that in vanilla TTT equipment items have an numeric id and this is the first thing I thought of. If I'd have to decide I's say `classname` is the most descriptive name for this.

Reply:
See this line in original TTT: https://github.com/Facepunch/garrysmod/blob/87aa68cb8da3d70e6e7265bc61ca222c55e3f8e5/garrysmod/gamemodes/terrortown/gamemode/equip_items_shd.lua#L80

Reply:
I believe that id was introduced by @Alf21 to make getting the classname shorter. But as we tend to give descriptive names, maybe we should instead try to get rid of id instead.

Reply:
I like ID the most tbh because it fits both old and new code also we treat it as the unique identifier and it is not just a class which is rather far from being a word to identify a single element.

Reply:
But there is an ID in the old ttt Items which isn't this one

Reply:
Need to check the codebase then

Reply:
I still think ID is more fitting because in the items module it is also used as ID and class can be confused with the weapon classnames.

Reply:
Well, class is definitely wrong, which for me should only be handled for weapon and item tables.

But classname, id and name are all the same in ohr codebase. We habe that one function, that sets id and name to the classname.

Also garrysmod uses classname with weapons.GetStored and the give function.

So in general that does not make a big difference. But I will get rid of `cls` in favor of one of the three, we just gotta try to be consistent about it.

Reply:
I vote for classname, id sounds wrong for me